src/aap_eda/api/serializers/activation.py

```diff
@@ -126,18 +122,14 @@ def _handle_eda_credentials(
 
         data = substitute_variables(injectors, user_inputs)
 
-        if extra_var_obj:
-            existing_data = yaml.safe_load(extra_var_obj.extra_var)
+        if validated_data["extra_var"]:
+            existing_data = yaml.safe_load(validated_data["extra_var"])
```

Comment:
Should be `extra_vars`  (plural) so it's aligned with what the rest of the code expects. I get a HTTP 500 and a KeyError when doing a POST to `/credential-types/` something like:
```json
{
    "name": "QE Secret with injector",
    "namespace": "namespace",
    "kind": "kind",
    "description": "foo",
    "inputs": {
        "fields": [
            {
                "id": "injected_secret",
                "type": "string",
                "label": "Injected Secret",
                "secret": true
            }
        ]
    },
    "injectors": {
        "extra_vars": {
            "my_msg": "bla bla"
        }
    }
}
```